### ISSUE TITLE
Making review question boxes also touchable so that can we can swipe it to next question.

### ISSUE THREAD

**Issue by chaytanyasinha** [commons-app/apps-android-commons#3206]
Summary:
We should let the whole area of review cards which contains "Is this in-scope?","Is this under category correctly"which is there in the review section instead of the small dots pagination in the most bottom.
And I also feel like the Skip this image button should be in the extreme bottom.
Steps to reproduce the issue:
Just go to the review section and you may try swipe to the other question but for that you have to go to the dots pagination for moving to the next question which sometimes not working at the first go.
What did you expect the app to do, and what did you see instead?
App should be easy to be handled with swiping functions so that user can easily move to the next question.
And the Skip Image button should be in the bottom so that if the user is not knowing about the picture he/she can first check the image then take the decision of skipping the image or not

Device-Samsung S6  Android Version-8.0

Would you like to work on the issue?
Yes.
![WhatsApp Image 2019-11-09 at 22 28 01](https://user-images.githubusercontent.com/50200796/68883584-593d3280-0737-11ea-8f1b-1e78823993db.jpeg)
![WhatsApp Image 2019-11-09 at 22 28 56](https://user-images.githubusercontent.com/50200796/68883585-59d5c900-0737-11ea-8157-9fc0bf59def7.jpeg)

**Comment by neslihanturan** [comment]
Thanks for your suggestion @chaytanyasinha :) as a user i wouldnt expect to go to next question when I click on the the question itself. I am not sure if it is a needed improvement. Please everyone join the discussion

**Comment by chaytanyasinha** [comment]
Correct ma'am but I just want to say that if an user can go to thd next question by sliding through the dots,then sliding the questions will much easier.

**Comment by neslihanturan** [comment]
Sorry I thought you were talking about tapping questions not sliding them. Actually I am not really sure what does screenshots displays and what the issue really is. I have read the issue description carefully but would be great if you can edit it to make it more clear. And it also would be great if you can note if the screenshots indicates the current situation or the situation after the improvement you suggested.

**Comment by neslihanturan** [comment]
And a friendly reminder, we are a team here so no need to say ma"am. You can say friend or tag our name instead:)

**Comment by chaytanyasinha** [comment]
@neslihanturan I made the changes in the issue description

**Comment by chaytanyasinha** [comment]
So should I work on it ?@neslihanturan

**Comment by neslihanturan** [comment]
Well, recently I get a suggestion from a user which says "skip this question" button can be beneficial. I think this issue seems related. What does everyone else think?

**Comment by chaytanyasinha** [comment]
@neslihanturan your suggestions are most welcome and If you want I can do whatever changes are required in the section.

**Comment by neslihanturan** [comment]
Well, currently seems fine button does nothing, so it just skips the question. It is basically the same thing with skip this question button. So I am not sure if it is exactly needed. But it is said to me that when the user is not 100% sure what does this question means, it is hard to press seems fine button. ınstead they prefer a skip this question button. 

This issue needs to be discussed more @chaytanyasinha

**Comment by chaytanyasinha** [comment]
Yes @neslihanturan , I also think this is an issue which is to be discussed and as much I know there is no skip this question button available and for going to the next question we have the dots in bottom which is the area which can touched to slide which is not that comfortable to move to next question instead of any kind button.So I think the whole card of question should be able to slide and we can keep the skip image button in the most bottom if a person is not able to review the question.
![WhatsApp Image 2019-11-23 at 22 42 41](https://user-images.githubusercontent.com/50200796/69484466-7c639280-0e59-11ea-9eda-5f76adedf9df.jpeg)

**Comment by chaytanyasinha** [comment]
@neslihanturan So what do you think?

**Comment by animeshk08** [comment]
Hi, is this issue still being discussed? @neslihanturan  I have also faced this issue. Please see the below screenshot for better understanding. As you can see from the touches on the screen. The questions are not swipeable in the text region. However, on swiping across the blue region(3 white dots), the questions do get skipped and that too both ways. However, this functionality is not very intuitive and is pretty much hidden. Also, I could not find any ```skip this question``` button. If this needs to be solved can I work on this?

<img src="https://user-images.githubusercontent.com/32506591/73784409-6eb47c80-47bb-11ea-8a8e-45416d575342.gif" width="400">

**Comment by animeshk08** [comment]
any updates? @neslihanturan
